.setup/apache/submitty.conf

```diff
@@ -40,6 +40,12 @@
         Require all granted
 
         RewriteEngine On
+        
+        # Routing Flexibility: Removes the trailing slash if it is supplied
+        # Example: /courses/f21/csci1200/ -> /courses/f21/csci1200
+        RewriteCond %{REQUEST_FILENAME} !-d
+        RewriteCond %{REQUEST_URI} /(.*)/$
+        RewriteRule ^ /%1 [R=301,L]
```

Comment:
Hm, this actually seems to break git. I'm not sure why it seems git auto-adds a trailing slash, so we may need to special case this so to ignore any URIs that include a `/git/` path?

Reply:
> Given the redirect that's currently happening, I wonder if we could put in place a patch to `site/public/index.php` to have similar behavior on the redirect?

At that point we should just follow [the guide](https://symfony.com/doc/3.3/routing/redirect_trailing_slash.html) refereed in https://github.com/Submitty/Submitty/issues/7253#issuecomment-977482083 or make a 404 handler if applicable

Reply:
That'd also be fine.

Reply:
okay the guide I've found previously refers to 3.x version of symfony, apparently this [should be enabled](https://symfony.com/doc/current/routing.html#redirecting-urls-with-trailing-slashes) in newer version ([>4.0](https://symfony.com/doc/4.1/routing/redirect_trailing_slash.html)) by default out of the box? (also mentioned in https://github.com/symfony/symfony/issues/39875). I wonder why this doesn't it work for us.

Reply:
Weird, now the git tests are passing. I wonder if it was just a temporary hiccup? This PR might be good to go without any further changes, though it still might be better to move this into the PHP code if it's not too much trouble as then don't need to have upgrade instructions for sysadmins...

Reply:
will implementing this behavior in `WebRouter.php` a cleaner alternative?

Reply:
I think so?